tests/python_tests/test_generate_api.py

```diff
@@ -711,32 +707,3 @@ def test_chat_1(model_descr, generation_config):
         print(f'ov_output: {chat_history_ov}')
     assert chat_history_ov == chat_history_hf
     pipe.generate('你好！ 你好嗎？', max_new_tokens=20)
-
-
-@pytest.mark.skip(reason="probably both models ov + hf doesn't fit to memory")
-@pytest.mark.precommit
-@pytest.mark.skipif(sys.platform.startswith("win"), reason="probably not enough space for this model on Win")
-def test_left_pad():
```

Comment:
Test is redundant as `pad_left` implementation tested implicitly on batched prompts.